docs/02_database/01_fields-tables-views/02_views/01_views-basics.mdx

```diff
@@ -123,4 +125,37 @@ You can override the name of a field using various operators:
       INSTRUMENT.NAME withPrefix INSTRUMENT
       INSTRUMENT.CURRENCY_ID withAlias "CURRENCY"
 ```
+## Backwards joins
+Backwards joins are important if your view is used in any real-time activity, for example, if you are going to stream data from the view or if you have [consolidators](/server/consolidator/introduction/) monitoring the view.
+
+By default, the platform (specifically, the [Update Queue](/operations/update-queue/overview/)) only listens for changes on the root table of the view.
```

Comment:
Maybe again focus on the joined data here. We could use the example of an instrument table joining onto a price table and a market table. We want the join to the price table to be backwards but not the market table. 

This is because the pricing table is updated on an ongoing basis and the market table is mostly static data.